zerver/models.py

```diff
@@ -3443,15 +3443,15 @@ def is_stream_admin(self) -> bool:
     # * "is_muted" often needs to be copied to not "in_home_view" for
     #   backwards-compatibility.
     API_FIELDS = [
-        "color",
-        "is_muted",
-        "pin_to_top",
         "audible_notifications",
+        "color",
         "desktop_notifications",
         "email_notifications",
+        "is_muted",
+        "pin_to_top",
         "push_notifications",
-        "wildcard_mentions_notify",
         "role",
+        "wildcard_mentions_notify",
```

Comment:
Are these `models.py` changes just sorting the list?  Would be a nice prep commit.